apps/contact/views.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib import messages
+from django.core.mail import EmailMessage
+from django.shortcuts import redirect, render_to_response
+from django.template import RequestContext
+
+from apps.contact.forms import ContactForm
+
+
+# Index page
+def index(request):
+    context = {'form': ContactForm}
+    return render_to_response('contact/index.html', context, context_instance=RequestContext(request))
+
+
+def contact_submit(request):
+    if request.method == 'POST':
+        form = ContactForm(request.POST)
+        if form.is_valid():
+            if form.cleaned_data['contact_receiver'] == '1':
+                choice = ['hovedstyret@online.ntnu.no']
+            else:
+                choice = ['dotkom@online.ntnu.no']
+
+            subject = '[Kontakt]' + form.cleaned_data['contact_name'] + 'har kontaktet dere gjennom ow4'
```

Comment:
Use Python string substitution for this, looks much cleaner (also, by using concatenation by `+` here, you don't have spaces between the strings, so it'll be `"[Kontakt]Nameharkontaktet dere gjennom ow4"`.

Like this: `[Kontakt] {name} …`.